evalml/tests/component_tests/test_components.py

```diff
@@ -1018,9 +963,9 @@ def test_components_can_be_used_for_partial_dependence_fast_mode():
     # Expected number is hardcoded so that this test will fail when new components are added
     # It should be len(all_native_components) - num_invalid_for_pd_fast_mode
     if ProphetRegressor not in all_native_components:
-        expected_num_valid_for_pd_fast_mode = 65
+        expected_num_valid_for_pd_fast_mode = 62
```

Comment:
codecov says this line isn't covered by test (and the check for codecov fails as well) if I don't include line 947-948.